SDWebImageHEIFCoder.podspec

```diff
@@ -60,17 +57,14 @@ Which is built based on the open-sourced libheif codec.
 
   # HEIF Encoding need libx265
   s.subspec 'libx265' do |ss|
+    ss.dependency 'libx265'
     ss.dependency 'SDWebImageHEIFCoder/libheif'
-    ss.source_files = 'Vendors/include/libx265/*.{h}', 'Vendors/libheif/libheif/heif_encoder_x265.{h,c,cc}'
-    ss.public_header_files = 'Vendors/include/libx265/*.{h}'
-    ss.osx.vendored_libraries = 'Vendors/libx265/macOS/libx265.a'
-    ss.ios.vendored_libraries = 'Vendors/libx265/iOS/libx265.a'
-    ss.tvos.vendored_libraries = 'Vendors/libx265/tvOS/libx265.a'
-    ss.watchos.vendored_libraries = 'Vendors/libx265/watchOS/libx265.a'
-    ss.preserve_path = 'Vendors/include'
+    ss.source_files = 'Vendors/libheif/libheif/heif_encoder_x265.{h,c,cc}'
+    ss.private_header_files = 'Vendors/libheif/libheif/heif_encoder_x265.h'
+    ss.preserve_path = 'Vendors'
     ss.xcconfig = {
       'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) HAVE_X265=1',
-      'HEADER_SEARCH_PATHS' => '$(inherited) ${PODS_ROOT}/SDWebImageHEIFCoder/Vendors/include'
+      'HEADER_SEARCH_PATHS' => '$(inherited) ${PODS_ROOT}/SDWebImageHEIFCoder/Vendors/include ${PODS_ROOT}/libx265/source/'
     }
```

Comment:
This `${PODS_ROOT}/libx265/source/` is here to fix suck of `libheif/heif_encoder_x265.cc`, which use x265 like this:

```c
extern "C" {
#include <x265.h>
}
```